### PR TITLE
Allow Hiding of Docker Command Messages

### PR DESCRIPTION
**Pull request by LaurenceWarne**
Hi, really liking the new async stuff, this is just a small PR which allows you to inhibit the messaging of run docker commands:

```elisp
(setq docker-message-commands nil)
```

The default it `t` so the default behaviour does not change.

Thanks!

### CODE REVIEW

File: docker-process.el [hunk 1]
```diff
@@ -36,6 +36,11 @@
   :group 'docker
   :type 'boolean)
 
+(defcustom docker-message-commands t
```
Comment (LaurenceWarne) [inline docker-process.el:39]: Naming vetos welcome

File: docker-process.el [hunk 2]
```diff
@@ -91,7 +96,8 @@
       (aio-resolve promise
                    (lambda ()
                      (message nil)
-                     (message "Finished: %s" (process-name process))
+                     (when docker-message-commands
```
Comment (LaurenceWarne) [inline docker-process.el:99]: Out of curiosity, what does `(message nil)` above/below do here?

Comment (Silex) [reply docker-process.el:99]: It hides the previous message otherwise the minibuffer spans multiple lines

File: docker-process.el (hunk 1 above)
Comment (Silex) [reply docker-process.el:39]: Please rename to `docker-show-messages`.

**Comment by LaurenceWarne** [comment]
> Be aware that this will be removed once I find the time to work on putting these messages in some magit-section like the `$` buffer in magit.

That would be a nice feature (though a quick of read of `magit-process.el` looks to me `magit-process-mode` is somewhat coupled with magit :cry:).  If that's the case I'd be inclined to close this, though I've pushed a commit in case you want to merge.

**Comment by Silex** [comment]
Thanks :+1:

**Pull request merged**